adodbapi/remote/server.py

```diff
@@ -97,7 +92,7 @@
 Pyro4.config.PREFER_IP_VERSION = 0  # allow system to prefer IPv6
 Pyro4.config.SERIALIZERS_ACCEPTED = set(
     ["serpent", "pickle"]
-)  # change when Py2.5 retired
+)  # TODO: change when Py2.5 retired
```

Comment:
What's the change needed here ?

Reply:
> The simplest answer here would be to delete the entire "remote" directory.
I don't think that the facility has ever been used by anyone other than
myself, and the use case for that is now very outdated.

A quick search I didn't find any reference to the remote directory either. I did find some to the remote module though, but only in tests. 

> I am in favor of dropping the entire subsystem as deprecated.

Do you also mean `adodbapi/remote.py` ? (I've left it in for now since you only mentioned deleting the directory)

Reply:
Cleaning it up properly wasn't as simple as just deleting 3 files so I created https://github.com/mhammond/pywin32/pull/1990 for that.